fix(curriculum) fixed vague wording as specified in issue #55373 (#55379)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-dom-manipulation-by-building-a-rock-paper-scissors-game/663d5697d80fef0eea026672.md

**Tips**

> - Set the display property of `resetGameBtn` as block.
> 
> - You can use the `el.style.display` property to show the `resetGameBtn` button and hide the `optionsContainer`. Here `el` refers to any element in reference.

Reviewer: ```suggestion
Use the `style.display` property on an element, with the value `"block"` or `"none"`, to show or hide the element.
```

I think we can generalize this hint. They already have been told what elements to show/hide.

Author: sounds good to me
